src/Microsoft.VisualBasic/ref/Microsoft.VisualBasic.cs

```diff
@@ -119,6 +119,30 @@ public sealed partial class Strings
         public static char ChrW(int CharCode) { throw null; }
         public static string[] Filter(object[] Source, string Match, bool Include = true, [Microsoft.VisualBasic.CompilerServices.OptionCompareAttribute] CompareMethod Compare = CompareMethod.Binary) { throw null; }
         public static string[] Filter(string[] Source, string Match, bool Include = true, [Microsoft.VisualBasic.CompilerServices.OptionCompareAttribute] CompareMethod Compare = CompareMethod.Binary) { throw null; }
+        public static int Len(bool Expression) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public static int Len(sbyte Expression) { throw null; }
+        public static int Len(byte Expression) { throw null; }
+        public static int Len(short Expression) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public static int Len(ushort Expression) { throw null; }
+        public static int Len(int Expression) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public static int Len(uint Expression) { throw null; }
+        public static int Len(long Expression) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public static int Len(ulong Expression) { throw null; }
+        public static int Len(decimal Expression) { throw null; }
+        public static int Len(float Expression) { throw null; }
+        public static int Len(double Expression) { throw null; }
+        public static int Len(System.DateTime Expression) { throw null; }
+        public static int Len(char Expression) { throw null; }
+        public static int Len(string Expression) { throw null; }
+        // QUESTION:
+        // Why has this method an attribute in the reference file https://gist.github.com/danmosemsft/88da57da7f11a9a3345abb5ec508d5e7
+        // and no attribute in https://raw.githubusercontent.com/Microsoft/referencesource/master/Microsoft.VisualBasic/runtime/msvbalib/Strings.vb?
+        [System.Security.SecuritySafeCriticalAttribute]
```

Comment:
.NET Core does not respect these System.Security.* attributes so we removed them when porting. You can remove it here.

Reply:
Removed.